contracts/opsuccinctfdgconfig.baklava.json

```diff
@@ -0,0 +1,30 @@
+{
+  "aggregationVkey": "0x000ce954cedc18b3dbb347e363d59a6d993d5e7e4a11df76864599bf41a72ceb",
```

Comment:
Can we add some explanation to the fields ? eg aggregationVkey rangeVkeyCommitment ...

Reply:
Added: https://github.com/celo-org/op-succinct/blob/21fd73552dd228dc629722fc1701ccaa7230b73f/README.md#celo-remarks

Resolving 😁